Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionForcePointFirewall.yaml

```diff
@@ -0,0 +1,473 @@
+                    EventEndTime = todatetime(ReceiptTime),
+                    boolHasIP = iif(indexof_regex(requestURL,@"(\d+.\d+.\d+.\d+)") == -1, false, true),
+                    boolHasPort = iif(requestURL has ':', true, false)
+            | extend 
+                     NetworkRuleName = coalesce(DeviceCustomString1, //RuleId
+                                                DeviceCustomString2), //NatRuleId
+                     ThreatId = coalesce(DeviceCustomString3, //VulnerabilityReferences
+                                        DeviceCustomString4), //VirusId
+                     DstDomain = case(
+                                    boolHasPort and not(boolHasIP), tostring(split(requestURL,':',0)[0]),
+                                    not(boolHasIP), requestURL,
+                                    ""
+                                ),
+                     DstIpAddr = case(
+                                    isnotempty(DstIpAddr), DstIpAddr,
+                                    not(boolHasIP), requestURL,
+                                    ""
+                                )
+            | extend
+                     DvcIdType = case(
+                                    isnotempty(DvcId), "Other",
+                                    ""
+                                ),
+                     DvcDomainType = case (
+                                    isnotempty(DvcDomain), "FQDN",
+                                    ""
+                                ),
+                     DstDomainType = case(
+                                    isnotempty(DstDomain), "FQDN",
+                                    ""  
+                                ),
+                     DstPortNumber = case(
+                                    isnotempty(DstPortNumber), DstPortNumber,
+                                    ApplicationProtocol startswith "TCP", toint(split(ApplicationProtocol,'/')[1]),
+                                    ApplicationProtocol startswith "UDP", toint(split(ApplicationProtocol,'/')[1]),
+                                    boolHasPort, toint(split(requestURL,':',0)[1]),
+                                    int(null)
+                            ),
+                     AdditionalFields = pack("RequestMethod",iff(isnotempty(RequestMethod) and RequestMethod != "UNKNOWN", RequestMethod, ""),
+                                             "Activity",Activity),
+                     ThreatName = case(isnotempty(ThreatId),DeviceEventCategory,
+                                      ""),
+                     DstAppName = case(DestinationServiceName in~ ("Generic-Web-HTTP","Application-Unknown","Unknown-Encrypted-Application"), "",
+                                        DestinationServiceName),
+                     DvcIpAddr = coalesce(DvcIpAddr,DeviceName)
+            | extend
+                    EventSeverity = case(isnotempty(EventSeverity),EventSeverity,
```

Comment:
Is the original EventSeverity normalized?

Reply:
I have populated EventSeverity using the lookup tables. This will always have normalized values